tests/storage/outofprocess_test.py

```diff
@@ -53,19 +54,15 @@ def oop_cleanup():
 
 @pytest.fixture(
     params=[
-        userstorage.PATHS["mount-512"],
-        userstorage.PATHS["mount-4k"],
+        BACKENDS["mount-512"],
+        BACKENDS["mount-4k"],
     ],
     ids=str,
 )
 def user_mount(request):
-    storage = request.param
-    if not storage.exists():
-        pytest.xfail("{} storage not available".format(storage.name))
-    tmpdir = os.path.join(storage.path, "tmp")
-    os.mkdir(tmpdir)
-    yield tmpdir
-    shutil.rmtree(tmpdir)
+    with request.param:
+        with request.param.tmpdir() as tmpdir:
+            yield tmpdir
```

Comment:
It would be more consistent and more clear like this:

    backend = request.param
    with backend:
        with backend.tmpdir() as tmpdir:
            yield tmpdir